### ISSUE TITLE
Unable to make it work

### ISSUE THREAD

**Issue by janluke** [storax/jinjaapidoc#10]
I'm using Sphinx 1.8.5. The build succeed but no api document is generated (nor the base folder). No error or warning, apart from that due to missing api reference files:

WARNING: toctree contains reference to nonexisting document 'api/<package-name>`

(I replaced the real package name with <package-name>)

Am I missing something? 

```
this_dir = os.path.abspath('.')
src_dir = os.path.abspath('../src')
sys.path.insert(0, src_dir)

extensions = [
    'jinjaapidoc',
    'sphinx.ext.napoleon',
    'sphinx.ext.autodoc',
    'sphinx.ext.autosummary',
    'sphinx.ext.intersphinx',
    'sphinx.ext.coverage',
    'sphinx.ext.ifconfig',
    'sphinx.ext.viewcode',
    'sphinx_autodoc_typehints',
    'IPython.sphinxext.ipython_console_highlighting',
    'IPython.sphinxext.ipython_directive'
]

templates_path = ['_templates']
source_suffix = '.rst'
master_doc = 'index'

exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']

# jinja-apidoc
autosummary_generate = True
jinjaapidoc_src = src_dir
jinjaapi_outputdir = os.path.join(this_dir, 'api')
```

**Comment by storax** [comment]
Hm, this package wasn't compatible with newer Sphinx versions. I've released 0.5.0 and tested it on Sphinx 1.8.5 and 2.0.1. Could you try upgrading and check if that fixes the issue, please?
Let me know if it works or not.

**Comment by janluke** [comment]
I managed to make it work but I have no idea what made the difference lol. It wasn't the upgrade. It didn't work even after the upgrade. In fact, I had already tried the old package with older versions of Sphinx (<1.8).

OT question: in your conf.py you set `jinjaapi_srcdir` to `"src"`. I used the same template for my project, still I had to provide `"src/<package_name>"`. Why is that?
